{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"description"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/axpcore\/axperror\/description"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AXPCore\/documentation\/AXPCore\/AXPError\/description","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"A textual description of the error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"description"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"title":"description","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:7AXPCore8AXPErrorO11descriptionSSvp","modules":[{"name":"AXPCore"}]},"hierarchy":{"paths":[["doc:\/\/AXPCore\/documentation\/AXPCore","doc:\/\/AXPCore\/documentation\/AXPCore\/AXPError"]]},"references":{"doc://AXPCore/documentation/AXPCore/AXPError/description":{"role":"symbol","title":"description","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"description"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"A textual description of the error."}],"identifier":"doc:\/\/AXPCore\/documentation\/AXPCore\/AXPError\/description","kind":"symbol","type":"topic","url":"\/documentation\/axpcore\/axperror\/description"},"doc://AXPCore/documentation/AXPCore":{"role":"collection","title":"AXPCore","abstract":[{"type":"text","text":"The base module that establishes the session with Avaya Experience Platformâ„¢ (AXP) for the end user."}],"identifier":"doc:\/\/AXPCore\/documentation\/AXPCore","kind":"symbol","type":"topic","url":"\/documentation\/axpcore"},"doc://AXPCore/documentation/AXPCore/AXPError":{"role":"symbol","title":"AXPError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"AXPError"}],"abstract":[{"type":"text","text":"Error for the case where an operation failed due to an error contacting AXP"},{"type":"text","text":" "},{"type":"text","text":"across the network."}],"identifier":"doc:\/\/AXPCore\/documentation\/AXPCore\/AXPError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AXPError"}],"url":"\/documentation\/axpcore\/axperror"}}}